src/Persistence/PaYaml/Models/SchemaV3/ScreenInstance.cs

```diff
@@ -10,4 +10,6 @@ public record ScreenInstance : IPaControlInstanceContainer
     public NamedObjectMapping<PFxExpressionYaml>? Properties { get; init; }
 
     public NamedObjectSequence<ControlInstance>? Children { get; init; }
+
+    public int? DisplayOrder { get; init; }
```

Comment:
place this property before the `Properties` property. as this controls how the yaml gets serialized.